Formula/portablegl.rb

```diff
@@ -44,6 +44,9 @@ def install
   end
 
   test do
+    # Tests require PNG images to be byte-for-byte identical,
+    # which may be broken by -march=native.
+    ENV.remove_from_cflags "-march=native"
```

Comment:
Yeah, I knew the PNGs have to be pixel for pixel exact for the test to pass.  It was why I thought it could be an endian issue on the mac except then all the tests would fail.  And besides I'd think since I'm comparing the pixels not the actual file contents that it wouldn't matter since stb_image says it comes in as rgba in memory period.

I've heard of -march=native but never seen it used before, for
[good reason apparently](https://stackoverflow.com/questions/52653025/why-is-march-native-used-so-rarely)

So, all tests pass now and I can create the tag?

Reply:
This looks related: https://stackoverflow.com/questions/69971612/how-does-march-native-affect-floating-point-accuracy

> So, all tests pass now and I can create the tag?

Looks good to me so far. So, yes please.

Reply:
Done!